tla-types/src/main/scala/at/forsyte/apalache/tla/typecomp/BuilderUtil.scala

```diff
@@ -66,11 +66,11 @@ object BuilderUtil {
     )(variable: TBuilderInstruction,
       expr: TBuilderInstruction): TBuilderInstruction = for {
     exprEx <- expr
-    boundAfterExpr <- allBound // variable may not appear as bound in expr
+    boundAfterExpr <- getAllBound // variable may not appear as bound in expr
```

Comment:
Much clearer, thank you!

Just to confirm, the call to the `variable` instruction below doesn't update the state?

Reply:
It does, they all do.